pkg/test/resourcefixture/testdata/basic/tags/v1beta1/tagstagvalue/tagvalueproject/_http.log

```diff
@@ -148,7 +148,7 @@ X-Xss-Protection: 0
     "createTime": "2024-04-01T12:34:56.123456Z",
     "description": "For valuename resources.",
     "etag": "abcdef0123A=",
-    "name": "tagValues/${tagValueID}",
+    "name": "tagValues/${tagValueId}",
```

Comment:
Should these be reverted?

Reply:
I don't believe so.  We've been inconsistent in our normalization, but ID seems to be much more common than Id.  The reason for the change is that we actually change tagValues in two places, I think I will actually just remove the second to avoid confusion.